README.md

```diff
@@ -1,46 +1,429 @@
+  - [Working with Arc.js Scripts](#working-with-arcjs-scripts)
+  - [Deploying to Other Testnets](#deploying-to-other-testnets)
+  - [Running Against a Ganache Database](#running-against-a-ganache-database)
+  - [Contribute to Arc.js](#contribute-to-arcjs)
+  - [Security](#security)
+  - [License](#license)
 
-DAOstack is a widely open collaboration to build the basic framework for [Decentralized Autonomous Organizations](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization) (DAO) through bootstrap, or dogfooding.
+## Getting Started
 
+### Installation
 
-# Contributing
+Ensure that [NodeJS](http://nodejs.org/), v9.4.0 or greater, is installed.
 
-Contributions and pull requests are very welcome. Join us on [Slack](daostack.slack.com).
+Install the npm package into your project:
 
-If you want to contribute to the code, check out  [CONTRIBUTING.md](CONTRIBUTING.md).
+```script
+npm install @daostack/arc.js --save
+```
+
+Now you can proceed with migrating contracts to a running testnet and configuring Arc.js in your application and environment.
+
+### Setting up a Testnet with Arc Contracts
+
+Arc.js runs against an Ethereum network where it assumes that the Arc contracts have been migrated.  Out of the box, Arc.js can find contracts migrated to the mainnet. But for testing, you will need to tell it to migrate the Arc contracts to a testnet of your choice.  You can do this by running a few Arc.js scripts.
+
+For example, to deploy contracts to a ganache testnet, run the following scripts from the root of your application:
+
+On Linux and others [until runAsync is fixed for Linux](http://github.com/daostack/arc.js/issues/64), run this in a separate window of your own creation:
+
+```script
+npm explore @daostack/arc.js -- npm start test.ganache.run
+```
+
+On Windows: 
+```script
+npm explore @daostack/arc.js -- npm start test.ganache.runAsync
+```
+
+Now migrate the Arc contracts to ganache:
+
+```script
+npm explore @daostack/arc.js -- npm start migrateContracts
+```
+
+Now when your app uses Arc.js, it will be running against ganache and the contracts you just migrated.
 
+For more on Arc.js scripts, see [Working with Arc.js Scripts](#working-with-arcjs-scripts).
 
-# Arc Configuration
-The default configuration settings for Arc can be found in config/default.json, they are:
+### Using the Arc.js Library in your Code
+
+#### Importing Arc.js
+
+Import everything from ArcJs as follows:
 
 ```javascript
+import * as ArcJs from '@daostack/arc.js';
+```
+
+#### Configuring Arc.js
+The default configuration settings for Arc.js can be found in its `config/default.json` file. A few  examples:
+
+```json
 {
-  "providerUrl": "http://localhost:8545",
-  "network": "kovan", // Options are 'homestead', 'ropsten', 'rinkeby', 'kovan'
-  "gasLimit": [a number like 6015000]
+  "providerUrl": "http://127.0.0.0:8545",
+  "network": "ganache",
```

Comment:
why options were omitted?

Reply:
no need for the example to set all of them. none were deleted here from the original

Reply:
I mean the comment: `// Options are 'homestead', 'ropsten', 'rinkeby', 'kovan'`

Reply:
Ah, the comment was deleted.  I wanted to keep that just in the documentation.